lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -191,3 +192,23 @@ def psfont(*args, **kwargs):
     ax.text(0.5, 0.5, 'hello')
     with tempfile.TemporaryFile() as tmpfile, pytest.raises(ValueError):
         fig.savefig(tmpfile, format='pdf')
+
+
+@needs_tex
+def test_failing_latex():
+    """Test failing latex subprocess call"""
+    _, path = tempfile.mkstemp(suffix="pdf")
+    matplotlib.rcParams['text.usetex'] = True
+
+    # This failes with "Double subscript"
+    plt.xlabel("$%f_2_2$" % np.random.random())
+    with pytest.raises(RuntimeError):
+        try:
+            plt.savefig(path)
+        except RuntimeError as exc:
+            output = str(exc)
+            assert "LaTeX was not able to process the following string:" \
+                   in output
+            assert "Double subscript." in output
```

Comment:
This extra `try`/`except` is not necessary; use the [result of the `pytest.raises` context manager](https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions) to assert the message.